pollen-collector/feedparser.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/mmcdole/gofeed"
+)
+
+func getTodaysPollenFeed() (*gofeed.Feed, error) {
+	fp := gofeed.NewParser()
+	feed, err := fp.ParseURL("http://www.dmi.dk/vejr/services/pollen-rss/")
```

Comment:
Should we use `https://` just for good measure? 🔒

Reply:
Great idea, but it seems like the new version of DMI that launched today has removed the pollen RSS feed. My lucky day.. :)